src/benchmark/tpch_benchmark.cpp

```diff
@@ -49,7 +49,7 @@ int main(int argc, char* argv[]) {
   cli_options.add_options()
     ("s,scale", "Database scale factor (1.0 ~ 1GB)", cxxopts::value<float>()->default_value("0.1"))
     ("q,queries", "Specify queries to run (comma-separated query ids, e.g. \"--queries 1,3,19\"), default is all", cxxopts::value<std::string>()) // NOLINT
-    ("use_prepared_statements", "Do not use prepared statements instead of random SQL strings", cxxopts::value<bool>()->default_value("true")); // NOLINT
+    ("use_prepared_statements", "Use prepared statements instead of random SQL strings", cxxopts::value<bool>()->default_value("false")); // NOLINT
```

Comment:
@mrzzzrm Do you plan to support this for TPC-H? It would be nice if this could be true by default again

Reply:
You mean for JOB?

Reply:
I'm sure this could be done (after this is merged, I suppose?)

Reply:
Argh. I clicked on the wrong line. I was thinking about the cache_binary_tables feature. Prepared statements is irrelevant because the JOB queries are non-random anyway and will be cached.

Reply:
I guess it's possible but we'd need to encode the scale factor into the table names somehow. `customer.0[.]01.bin`, for sf=0.01 for example.

Reply:
Sure